[*] Running ...
[*] Dataset loaded: E:\Projects\CO\reevo-main_1/problems/tsp_gls\dataset/train200_dataset.npy with 10 instances.
Traceback (most recent call last):
  File "E:\Projects\CO\reevo-main_1/problems/tsp_gls/eval.py", line 51, in <module>
    obj = solve(instance)
          ^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main_1/problems/tsp_gls/eval.py", line 21, in solve
    heu = heuristics(inst.distmat.copy())
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\Projects\CO\reevo-main_1\problems\tsp_gls\gpt.py", line 10, in heuristics_v2
    if i != j:
^^^^^^^^^^^^^^^
ValueError: could not broadcast input array from shape (199,) into shape (200,)
